Sources/NIOExtras/DebugInboundEventsHandler.swift

```diff
@@ -0,0 +1,95 @@
+import NIO
+
+public class DebugInboundEventsHandler: ChannelInboundHandler {
```

Comment:
Can you add a doc comment for this please?

Reply:
Sorry, but doc comments begin each line with the four character string: `/// ` (that is, three slashes and then a space).